src/main/java/org/opensearch/jdbc/ResultSetImpl.java

```diff
@@ -1071,14 +1075,22 @@ public Date getDate(String columnLabel, Calendar cal) throws SQLException {
 
     @Override
     public Time getTime(int columnIndex, Calendar cal) throws SQLException {
-        // TODO - implement?
-        return null;
+        log.debug(() -> logEntry("getTime (%d, %s)", columnIndex,
+                cal == null ? "null" : "Calendar TZ= " + cal.getTimeZone()));
+        checkCursorOperationPossible();
+        Time value = getTimeX(columnIndex, cal);
+        log.debug(() -> logExit("getTime", value));
```

Comment:
see comment: https://github.com/Bit-Quill/opensearch-project-sql/pull/192/files/8d15c35a0065037fdc1c5b18a86ae3260c284716

Reply:
Addressed in b1f8e06b72614396cb0f17378d1a50605046fb90 and b1de4b8ddfd214a347d5f9618ad362ac2022d19a